packages/components/tests/dummy/app/templates/components/tabs.hbs

```diff
@@ -194,5 +194,13 @@
 <section data-test-percy>
   <h3 class="dummy-h3" id="showcase"><a href="#showcase" class="dummy-link-section">§</a> Showcase</h3>
 
-  <Hds::Tabs />
+  <Hds::Tabs as |T|>
+    <T.Tab id="tab1" href="#section1">One</T.Tab>
+    <T.Tab id="tab2" href="#section2">Two</T.Tab>
+    <T.Tab id="tab3" href="#section3">Three</T.Tab>
+    
+    <T.TabPanel id="section1" aria-labelledby="tab1">Content one</T.TabPanel>
+    <T.TabPanel id="section2" aria-labelledby="tab2">Content two</T.TabPanel>
+    <T.TabPanel id="section3" aria-labelledby="tab3">Content three</T.TabPanel>
```

Comment:
I know for the final implementation we wouldn't want to require users to have to specify all these individual attributes/arguments. This is just for purposes of testing using child components before I start to write the JS to make a nicer API.